opensearch/backup.py

```diff
@@ -0,0 +1,179 @@
+import datetime
+import json
+import os
+import sys
+import time
+from typing import Optional
+from urllib.parse import quote
+
+import requests
+
+
+def env_bool(name: str, default: bool = False) -> bool:
+    value = os.environ.get(name)
+    if value is None:
+        return default
+    return value.lower() in ("1", "true", "yes", "on")
```

Comment:
I would clearly document in the README api contract that the accepted values are true or false and limit it to that.

You can throw an exception if the passed value is anything else.